ui/admin/app/components/form/host-set/index.hbs

```diff
@@ -52,12 +52,12 @@
       </field.errors>
     {{/if}}
   </form.textarea>
-
-  <form.actions
-    @disabled={{if @model.cannotSave @model.cannotSave}}
-    @enableEditText={{t 'actions.edit-form'}}
-    @submitText={{t 'actions.save'}}
-    @cancelText={{t 'actions.cancel'}}
-  />
-
+  {{#if (can 'save model' @model)}}
```

Comment:
I assume if the user has no capabilities to save the model, no buttons are show so the user have to navigate back?

Reply:
yes!